dashboard/app/reporting.go

```diff
@@ -75,7 +75,8 @@ func reportingPollBugs(c context.Context, typ string) []*dashapi.BugReport {
 
 func handleReportBug(c context.Context, typ string, state *ReportingState, bug *Bug) (
 	*dashapi.BugReport, error) {
-	reporting, bugReporting, _, _, _, err := needReport(c, typ, state, bug)
+	reporting, bugReporting, _, status, _, err := needReport(c, typ, state, bug)
+	log.Infof(c, "needReport: %s", status)
```

Comment:
Is this needed? Or debugging leftover?
If you want to keep this, it needs to print what bug it relates to. Otherwise it will be just hundreds unuseful log lines.

Reply:
Yes, it was just a leftover.